src/containers/ExhibitionDetail/index.jsx

```diff
@@ -1,11 +1,9 @@
 import React, { useState, useEffect } from 'react';
 import { useParams } from 'react-router-dom';
-import axios from 'axios';
+import axiosInstance from '../../axiosInstance';
```

Comment:
Since this is importing a custom module, please leave an empty line after the imports of external dependencies.